docs/README.md

```diff
@@ -185,7 +189,9 @@ This will start the media proxy in blocking mode and confirm that build was succ
 Run the media proxy:
 
 ```bash
-$ media_proxy
+media_proxy
+```
+```
 INFO: TCP Server listening on 0.0.0.0:8002
```

Comment:
I would add here (before INFO: TCP Server..." the line. sth like:
The result should look like this:

Reply:
Fixed.